bundles/org.openhab.binding.smarther/src/main/java/org/openhab/binding/smarther/internal/account/SmartherAccountService.java

```diff
@@ -0,0 +1,296 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.smarther.internal.account;
+
+import static org.openhab.binding.smarther.internal.SmartherBindingConstants.*;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.lang.invoke.MethodHandles;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Hashtable;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.binding.smarther.internal.api.dto.Notification;
+import org.openhab.binding.smarther.internal.api.dto.Sender;
+import org.openhab.binding.smarther.internal.api.exception.SmartherGatewayException;
+import org.openhab.binding.smarther.internal.util.StringUtil;
+import org.osgi.framework.BundleContext;
+import org.osgi.service.component.ComponentContext;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Deactivate;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.http.HttpService;
+import org.osgi.service.http.NamespaceException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@code SmartherAccountService} class manages the servlets and bind authorization servlet to Bridges.
+ *
+ * @author Fabio Possieri - Initial contribution
+ */
+@Component(service = SmartherAccountService.class, immediate = true, configurationPid = "binding.smarther.accountService")
+@NonNullByDefault
+public class SmartherAccountService {
+
+    private static final String TEMPLATE_PATH = "templates/";
+    private static final String IMAGE_PATH = "web";
+    private static final String TEMPLATE_APPLICATION = TEMPLATE_PATH + "application.html";
+    private static final String TEMPLATE_INDEX = TEMPLATE_PATH + "index.html";
+    private static final String ERROR_UKNOWN_BRIDGE = "Returned 'state' doesn't match any Bridges. Has the bridge been removed?";
+
+    private final Logger logger = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
```

Comment:
Whats with `MethodHandles.lookup().lookupClass()`?
Shouldn't this just be `SmartherAccountService.class`?

Reply:
Reflection I think I saw it in other bindings. I suppose they used it just to copy the logger row in new classes without bothering with the class name (so avoid potential bug in case they miss to).
No problem at all in changing it back to direct .class reference. Do you know it would be faster execution then or have any other bebefit ?